test-cases/tests/installation/a22-verify-rhmiconfig-maintenance-backup-validation.md

```diff
@@ -0,0 +1,33 @@
+---
+targets:
+- 2.3.0
+estimate: 30m
+---
+
+# A22 - Verify RHMIConfig maintenance and backup validation
+We have a RHMIConfig validation webhook, this test is to verify the maintenance and backup validation works as expected
+
+The expected value formats are:
+```yaml
+spec:
+  backup: 
+    applyOn : "HH:mm"
+  maintenance: 
+     applyFrom : "DDD HH:mm"
+```
+
+1. Add new values in correct format to ensure the validation works with no error
```

Comment:
I think being more explicit with the example values here makes sense. Even if it's an `oc patch` command. It will help speed up the testing and avoid misinterpretations of the test.